Cargo.toml

```diff
@@ -57,4 +57,4 @@ zint = { path = "zint", version = "=0.1.3" }
 
 [profile]
 dev = { panic = "abort"}
-release = { panic = "unwind" }
+release = { panic = "unwind" }
```

Comment:
plz create a new line after this line or there will be a EOF warning in github

same in other files